### PR TITLE
Logic

### PR DESCRIPTION
**Pull request by men32z**
- Required logic files
- Tested functions

### CODE REVIEW

File: src/logic/calculate.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import operate from './operate';
+
+const calculate = ({ total, next, operation }, buttonName) => {
+  const operators = ['÷', '+', '-', 'x', '%'];
+
+  if (operators.includes(buttonName)) {
+    return { total, next, operation: buttonName };
+  }
+
+  switch (buttonName) {
```
Comment (azdrenymeri) [inline src/logic/calculate.js:10]: You haven't covered the `case` when `buttonName` equals to a dot`(.)`

File: src/logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+/* eslint-disable-next-line */
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  switch (operation) {
+    case '÷':
+      return Big(numberOne).div(numberTwo).toString();
```
Comment (azdrenymeri) [inline src/logic/operate.js]: If `NumberTwo` is zero`(0)` then this line would throw a `DividedByZero` Error.
Kindly handle this error

**Review by azdrenymeri — CHANGES_REQUESTED** [review]
### Hello @men32z :wave: 
#### Good job on this milestone :+1: 
* Kindly next time add a more brief explanation on the description of the Pull Request.
* The project has two Readme.md files one called `README.md` and another one called `README.old.md` make sure to keep only one `README.md` file.
#### Kindly checkout the comments and implement the changes then add it for another review.

File: src/logic/operate.js (hunk 2 above)
Comment (men32z) [reply src/logic/operate.js]: Thanks! I will fix the errors

**Review by bellom — APPROVED** [review]
## Hi @men32z  :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**